cameo/api/designer.py

```diff
@@ -244,24 +251,23 @@ def build_results_data(self, strategy_designs, strategies, results, progress, of
         for i, strain_designs in enumerate(strategy_designs):
             strategy = strategies[i]
             _results = DataFrame(columns=results.columns, index=[j for j in range(len(strain_designs))])
-            manipulations, fitness, yields, target_flux, biomass = self.process_strain_designs(strain_designs,
-                                                                                               *strategy[1:])
+            manipulations, fitness, yields, target_flux, biomass = \
+                self.process_strain_designs(strain_designs, **strategy)
             for j, strain_design in enumerate(manipulations):
                 _results.loc[j, 'manipulations'] = strain_design
-                _results.loc[j, 'heterologous_pathway'] = strategy[2]
-            _results['host'] = strategy[0].name
-            _results['model'] = strategy[1].id
+                _results.loc[j, 'heterologous_pathway'] = strategy['pathway']
+            _results['host'] = strategy['host']
+            _results['model'] = strategy['model'].id
             _results['fitness'] = fitness
             _results['yield'] = yields
             _results['biomass'] = biomass
             _results['product'] = target_flux
             _results['method'] = "DifferentialFVA+PathwayFinder"
```

Comment:
Would be helpful if you could point out what I need to replace "DifferentialFVA+PathwayFinder" with?